### DISCUSSION TITLE
Feature Request: Building repos for different configurations

### DISCUSSION THREAD

**Discussion by B-R-P** [community/community#34778] [Repositories]
An option to build repo for different OS on click in cloud and download the executable. Instead of building on our own. It makes repos of early development easy for all to use.

**Comment by airtower-luna** [comment]
See: [Actions](https://docs.github.com/en/actions), specifically the [CI section](https://docs.github.com/en/actions/automating-builds-and-tests/about-continuous-integration). Upload the executables as artifacts after tests pasts and you have what you want. :wink:

**Reply by B-R-P** [reply]
Generating prebuild binaries for others (*not for contributors of repo*). A build button on repo page with different configurations like OS to choose.
